### PR TITLE
Use rockets python client

### PR DESCRIPTION
**Pull request by tribal-tec**

### CODE REVIEW

File: python/brayns/async_client.py [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+
+"""Client that connects to a remote running Brayns instance which provides the supported API."""
+
+import asyncio
+import rockets
+
+from .api_generator import build_api
+from .base import BaseClient
+from .utils import obtain_registry, convert_snapshot_response_to_PIL, SCHEMA_ENDPOINT
+from . import utils
```
Comment (rolandjitsu) [inline python/brayns/async_client.py]: Why not move what you have at `.` in `utils`?

Comment (rolandjitsu) [reply python/brayns/async_client.py]: Actually I see that it is there, so why import that one differently? Why not:
```py
from .utils import obtain_registry, convert_snapshot_response_to_PIL, SCHEMA_ENDPOINT, in_notebook
```

Comment (tribal-tec) [reply python/brayns/async_client.py]: That's a trap! The mocking otherwise does not work if the method is not used via its module name.

Comment (rolandjitsu) [reply python/brayns/async_client.py]: Got it 😄

File: python/brayns/base.py [hunk 2]
```diff
@@ -0,0 +1,242 @@
...
+    # pylint: disable=E1101
+    def __str__(self):
+        """Return a pretty-print on the currently connected Brayns instance."""
+        version = 'unknown'
+        if self.version:
+            version = '.'.join(str(x) for x in [self.version.major, self.version.minor,
+                                                self.version.patch])
+        return "Brayns version {0} running on {1}".format(version, self.http_url)
+
+    def set_colormap(self, palette, intensity=1, opacity=1, data_range=(0, 256)):
```
Comment (rdumusc) [inline python/brayns/base.py]: 0,256 -> 0,255? Or is the range interpreted as [0,256[ (not included) ?

File: python/brayns/base.py [hunk 3]
```diff
@@ -0,0 +1,242 @@
...
+            return show
+
+        setattr(self, 'show', function_builder())
+
+    def _add_animation_slider(self):  # pragma: no cover
+        """Add animation_slider() function for animation_parameters control."""
+        def function_builder():
+            """Wrapper for returning the animation_slider() function."""
+            def animation_slider():
+                """.Show slider to control animation"""
```
Comment (rdumusc) [inline python/brayns/base.py]: . misplaced :-)

File: python/brayns/client.py [hunk 4]
```diff
@@ -24,49 +24,49 @@
...
+        schemas = self.rockets_client.batch(requests)
+
+        schemas_dict = dict()
+        for request in requests:
+            # pylint: disable=protected-access,cell-var-from-loop
+            response = list(filter(lambda x, request_id=request.request_id(): x._id == request_id,
+                                   schemas))
+            # pylint: enable=protected-access,cell-var-from-loop
+            schemas_dict[request.params['endpoint']] = response[0].result
+        build_api(self, registry, schemas_dict)
```
Comment (rdumusc) [inline python/brayns/client.py]: code duplicated from AsyncClient? Move to base class?

**Comment by tribal-tec** [comment]
retest this please

**Comment by ppodhajski** [comment]
retest this please

**Comment by tribal-tec** [comment]
Need a review again actually

**Pull request merged**